cmd/minishift/cmd/root.go

```diff
@@ -179,3 +199,34 @@ func ensureConfigFileExists(configPath string) {
 		}
 	}
 }
+
+func performPostUpdateExecution() error {
+	fmt.Println("Binary has been recently updated.")
+	markerPath := filepath.Join(constants.Minipath, constants.UpdateMarkerFileName)
+	file, err := ioutil.ReadFile(markerPath)
+	if err != nil {
+		return err
+	}
+
+	var markerData UpdateMarker
+	var confirm = "y"
+
+	json.Unmarshal(file, &markerData)
+	if !markerData.Force {
+		fmt.Printf("Do you want to update the default addons? [y/N]: ")
```

Comment:
Please don't use the 'f' versions (Print_f_) of methods when you are not interpolating anything.